exercises/simple-cipher/canonical-data.json

```diff
@@ -0,0 +1,123 @@
+{
+  "exercise": "simple-cipher",
+  "version": "1.0.0",
+  "cases": [
+    {
+      "description": "Random key cipher",
+      "property": "random",
```

Comment:
I'm not sure, but I don't think this line actually does anything. The `property` key is normally used for specifying which method/property to test. As this is just a container for individual cases, I think it can be removed.